packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html

```diff
@@ -15,9 +15,7 @@ <h3 data-translate="telecom_pack_migration_doing"></h3>
             data-translate-values="{ fromOffer : $ctrl.process.pack.offerDescription, toOffer : $ctrl.process.selectedOffer.description }"
         ></p>
 
-        <div
-            data-ng-if="$ctrl.choosedAdditionalOptions.length > 0 || ($ctrl.process.selectedOffer.options.gtr_ovh && $ctrl.process.selectedOffer.options.gtr_ovh.selected)"
-        >
+        <div>
```

Comment:
**thought:** The following `div` looks like useless. Could be removed on separate Pull Request.